roles/openshift_service_catalog/tasks/remove.yml

```diff
@@ -3,9 +3,10 @@
   command: >
     oc delete apiservices.apiregistration.k8s.io/v1alpha1.servicecatalog.k8s.io --ignore-not-found -n kube-service-catalog
 
-- name: Remove Policy Binding
-  command: >
-    oc delete policybindings/kube-system:default -n kube-system --ignore-not-found
+# TODO: policybinding is not a resource type. what was the original intention of this?
```

Comment:
We could just remove this if it wouldn't actually do anything otherwise.

Reply:
We can remove it. My concern is that there may be something that should be done here. If we remove it, then we lose the marker that reminds us that something needs to be done. I haven't looked into it more deeply to understand what if anything needs to be done here.

Reply:
@deads2k @bparees do you happen to know if there is a concern with this hanging around after a removal?

Reply:
> @deads2k @bparees do you happen to know if there is a concern with this hanging around after a removal?

It won't exist at all in 3.7.  There's nothing to delete.

Reply:
That was my point. There is nothing to delete, but this was deleting something at some point. Presumably whatever it was deleting has been replaced by something else. What would it have been replaced with? And is there anything we need to do here to delete the replacement.

Reply:
> That was my point. There is nothing to delete, but this was deleting something at some point. Presumably whatever it was deleting has been replaced by something else. What would it have been replaced with? And is there anything we need to do here to delete the replacement.

It was not replaced with anything.  It did not have value.

Reply:
You are removing your rolebinding.rbac objects, right?  If you were counting on this having the side-effect of removing rolebindings (never recommended), then you have to delete rolebindings.rbac.

Reply:
I have removed the commented-out code that attempted to remove the policybinding. I have replaced it with code to delete the role-bindings via the templates created during install.